libs/sdk-ui-ext/src/internal/utils/colors.ts

```diff
@@ -128,6 +128,8 @@ export function getValidProperties(
                 return isUriRef(colorAssignment.headerItem.attributeHeader.ref)
                     ? colorAssignment.headerItem.attributeHeader.uri === id
                     : colorAssignment.headerItem.attributeHeader.identifier === id;
+            } else if (isColorDescriptor(colorAssignment.headerItem)) {
```

Comment:
I see there are some tests for this
```
    describe("getValidProperties", () => {
```
Please try adding one simple test case with the color descriptor for this condition if possible.

Reply:
I added the test. Thank you!